Extensions/retags.js

```diff
@@ -27,7 +27,7 @@ XKit.extensions.retags = {
 		var inner, $el;
 		ms.forEach(function(m){
 			$el = $(m.addedNodes);
-			inner = $el.find('.note:not(.ui_post_badge)');
```

Comment:
A reminder that retags covers: notes on the activity page, notes on some tumblr themes, notes directly on the dashboard, and the notes popup.